packages/react/src/UnderlineNav/UnderlineNav.tsx

```diff
@@ -338,7 +338,7 @@ export const UnderlineNav = forwardRef(
                   selectionVariant="single"
                   ref={containerRef}
                   id={disclosureWidgetId}
-                  sx={menuStyles}
+                  sx={menuStyles(containerRef.current, listRef.current)}
```

Comment:
I would suggest not to calculate the menu styles dynamically when it is not needed (i.e. when listRef's width is higher than the min width (192px))

We can maybe refactor this to be like below?

```
sx={listRef.current?.clientWidth && listRef.current.clientWidth >= 192 
? baseRules 
: menuStyles(containerRef.current, listRef.current)}
```

It would be good to take the 192 into const and reuse it (my example is bit of a pseudo code sorry 😄 )

Reply:
We will need to export the `baseRules` as well from `styles.ts`.

Reply:
Alright. Sounds good to me. I've moved the calculation to a ternary (like you suggested) outside of the function, as to not force the calculation for each render. 

Ive set the 192 to a const that I called `baseMenuMinWidth`. I hope that explains the variable well enough.